src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/SuperFinalizeCheck.java

```diff
@@ -39,6 +39,34 @@
  * <pre>
  * &lt;module name=&quot;SuperFinalize&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class A {
+ *  @Override
+ *      protected void finalize throws Throwable {
```

Comment:
`()` is missing.
Please make sure code is compilable

Reply:
Sure, I'll do it. Thank you so much